src/Analysis/Ast/Impl/Extensions/PythonModuleExtensions.cs

```diff
@@ -70,5 +74,39 @@ internal static void AddAstNode(this IPythonModule module, object o, Node n)
 
         internal static bool IsNonUserFile(this IPythonModule module) => module.ModuleType.IsNonUserFile();
         internal static bool IsCompiled(this IPythonModule module) => module.ModuleType.IsCompiled();
+
+        public static IEnumerable<string> GetSubmoduleNames(this IPythonModule module) {
+            var searchResult = module.Interpreter.ModuleResolution.CurrentPathResolver.FindImports(module.FilePath, Enumerable.Repeat(module.Name, 1), 0, true);
```

Comment:
You probably want to use `PathResolverSnapshot.GetModuleImportFromModuleName` here

Reply:
It was copied verbatim from `ImportCompletions`. Since no one actually uses it anymore, I'll remove it.